panel/src/panel/content.js

```diff
@@ -212,9 +239,9 @@ export default (panel) => {
 		async save(values, api) {
 			api ??= panel.view.props.api;
 
-			if (this.isCurrent(api) === true && this.isLocked(api) === true) {
-				throw new Error("Cannot save locked changes");
-			}
+			// if (this.isCurrent(api) === true && this.isLocked(api) === true) {
+			// 	throw new Error("Cannot save locked changes");
+			// }
```

Comment:
`Todo:` above we removed this code, so remove it here as well?

Reply:
I'm still not entirely sure about it to be honest. We could still keep the client-side checks in all methods and pass the view.props.lock object to the lockDialog method here.